Utilities/Constants.cs

```diff
@@ -25,6 +25,9 @@ public class Constants
         public static readonly string VEHICLE_SHIP = "Ship";
         public static readonly string VEHICLE_SRV = "SRV";
         public static readonly string VEHICLE_FIGHTER = "Fighter";
+        public static readonly string VEHICLE_LEGS = "On Foot";
```

Comment:
Possibly `VEHICLE_SUIT = "Suit";`? I'm in two minds here.

Reply:
I don't think "Suit" is appropriate since it's also the status when you're wandering around a station concourse without a suit.

Reply:
Au contraire, when wandering around the station concourse you are ALWAYS in a suit, even if it is just a vanilla flight suit.

Reply:
You can also be in a Maeverick, Artemis suit etc, but you are always in some sort of suit.

Reply:
Fair, you've convinced me to be open minded but I still don't see the virtue of `Suit` over `On Foot`?

Reply:
"Suit" is a noun like all the others, is all. Not a huge deal but it feels a bit more uniform.

Reply:
I dunno. You make a good point but I think that people will still expect "On Foot" or something similar.